index.html

```diff
@@ -992,7 +1016,10 @@ <h3>
           {{PerformanceResourceTiming/connectStart}},
           {{PerformanceResourceTiming/connectEnd}},
           {{PerformanceResourceTiming/requestStart}},
+          {{PerformanceResourceTiming/firstInterimResponseStart}},
           {{PerformanceResourceTiming/responseStart}},
+          {{PerformanceResourceTiming/responseHeadersEnd}},
+          {{PerformanceResourceTiming/responseBodyStart}},
```

Comment:
Thanks for making sure that the new timings require CORS. This is an essential requirement mentioned in the Chrome Security & Privacy intent review in order to avoid cross-origin leaks.